WindowsDevicePortalWrapper/WindowsDevicePortalWrapper.Shared/HoloLens/HolographicPerception.cs

```diff
@@ -57,10 +57,70 @@ public enum SimulationControlMode
         }
 
         /// <summary>
+        /// Enumeration defining the priority levels for the Holographic Perception Simulation control stream.
+        /// </summary>
+        public enum SimulationControlStreamPriority
+        {
+            /// <summary>
+            /// Low priority.
+            /// </summary>
+            Low = 0,
+
+            /// <summary>
+            /// Normal priority.
+            /// </summary>
+            Normal
+        }
+
+        /// <summary>
+        /// Creates a simulation control stream.
+        /// </summary>
+        /// <param name="priority">The control stream priority.</param>
+        /// <returns>PerceptionSimulationControlStreamId containing the identifier of the created stream.</returns>
+        /// <remarks>This method is only supported on HoloLens.</remarks>
+        public async Task<PerceptionSimulationControlStreamId> CreatePerceptionSimulationControlStream(SimulationControlStreamPriority priority)
```

Comment:
Is there a way to get all active streams? Not sure how this API works. Being able to create a stream but not get existing ones feels like an awkward edge case.